articles/azure-large-instances/what-is-azure-large-instances.md

```diff
@@ -0,0 +1,193 @@
+---
+title: What is Azure Large Instances?
+description: Provides an overview of Azure Large Instances.
+author: jjaygbay1
+ms.author: jacobjaygbay
+ms.service: baremetal-infrastructure
+ms.custom: references_regions
+ms.topic: conceptual
+ms.date: 06/01/2023
+---
+
+#  What is Azure Large Instances?
+
+Microsoft Azure offers a cloud infrastructure with a wide range of integrated cloud services to meet your business needs. 
+In some cases, though, you may need to run services on Azure large servers without a virtualization layer.
+You may need root access and control over the operating system (OS).
+To meet this need, Azure offers Azure Large Instances (ALI) for several high-value, mission-critical applications.
+
+Azure Large Instances is comprised of dedicated large instances (compute instances). It features:
+
+* High-performance storage appropriate to the application (Fiber Channel).
+Storage can also be shared across ALI to enable features like scale-out clusters or high availability pairs with failed-node-fencing capability. 
+
+* A set of function-specific virtual LANs (VLANs) in an isolated environment.
+
+This environment also has special VLANs you can access if you're running virtual machines (VMs) on one or more Azure Virtual Networks (VNets) in your Azure subscription.
+The entire environment is represented as a resource group in that subscription.
+
+A large set of ALI SKUs is available with Optane memory.
+Azure offers the largest range of Azure Large Instances in a hyperscale cloud.
+
+## Why ALI?
+
+Some workloads in the enterprise consist of technologies that just aren't designed to run in a typical virtualized cloud setting.
+They require special architecture, certified hardware, or extraordinarily large sizes.
+Although those technologies have the most sophisticated data protection and business continuity features, those features aren't built for the virtualized cloud.
+They're more sensitive to latencies and noisy neighbors and require more control over change management and maintenance activity.
+
+ALI is built, certified, and tested for a select set of such applications.
+Azure was the first to offer such solutions and has since led with the largest portfolio and most sophisticated systems.
+
+## ALI benefits
+
+ALI is intended for critical workloads that require certification to run your enterprise applications. 
+Azure Large Instances implementations are dedicated only to you, and you'll have full access (root access) to the operating system (OS).
+You manage OS and application installation according to your needs.
+For security, the instances are provisioned within your Azure Virtual Network (VNet) with no internet connectivity.
```

Comment:
```suggestion
For security, the instances are provisioned within your Azure Virtual Network (VNet) with no internet connectivity.If you need access to the internet then you need to setup an internet proxy service.
```
Added a quick update above Cc: @suzizuber

Reply:
This issue is resolved